packages/core/src/use-media-in-timeline.ts

```diff
@@ -58,23 +58,27 @@ export const useMediaInTimeline = ({
 	}, [duration, startsAt, volume, mediaVolume]);
 
 	useEffect(() => {
-		if (!mediaRef.current) {
+		const tagName = mediaType === 'audio' ? '<Audio>' : '<Video>';
+
+		if (!mediaRef.current || !hasMetadata) {
 			return;
```

Comment:
We are silently returning if no metadata, and then we are not checking for `currentSrc` in the next statement. Should we swap the order of these statements?